src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -321,6 +325,7 @@ class WorkflowViewContent extends Component {
       style: { padding: `1rem ${sideMargin} 0`, flex: 1, minHeight: 500 },
       activeStyle: { backgroundColor: colors.blue[3], cursor: 'copy' },
       ref: this.uploader,
+      onDropRejected: () => reportError('Error uploading inputs', 'The selected file is not a json file. To import inputs for this tool, upload a file with a .json extension.'),
```

Comment:
Maybe make the short text 'Only JSON files can be uploaded' or something?

Reply:
How about the short text as `Not valid JSON`: `The selected file is not a json file. To import inputs for this tool, upload a file with a .json extension.` ?